### PR TITLE
Modus styling issues

### PR DESCRIPTION
**Pull request by amosekr**
Fixed - If TMDropdown and TMInput TitleText is empty or null, empty space is displaying in the top of the field
Fixed - ControlLabelStyle is not applied for ControlLabel, Changed the ControlLabel style name to DefaultControlLabelStyle, 
Fixed - Helper text color change
Fixed - https://github.com/trimble-oss/modus-mobile-maui-components/issues/415 - Modus dropdown popup is not displaying properly in android
Fixed - ControlLabel and * aligenment not matched

### CODE REVIEW

**Comment by rthanga1** [comment]
@Sofiya-kumar Can you too review this one?

**Comment by Sofiya-kumar** [comment]
> @Sofiya-kumar Can you too review this one?

Sure @rthanga1

File: Trimble.Modus.Components/Controls/Popup/Pages/PopupPage.cs [hunk 1]
```diff
@@ -320,12 +320,12 @@ private void OnSizeChanged(object sender, EventArgs e)
...
                     case Enums.ModalPosition.Top:
                         translationY = loc.Top - _popupHeight - loc.Height / 2;
-                        translationX = loc.Center.X - _popupWidth / 2;
+                        translationX = loc.X;
                         break;
 
                     case Enums.ModalPosition.Bottom:
                         translationY = loc.Bottom - _popupHeight + loc.Height / 2;
-                        translationX = loc.Center.X - _popupWidth / 2;
+                        translationX = loc.X;
```
Comment (Sofiya-kumar) [inline Trimble.Modus.Components/Controls/Popup/Pages/PopupPage.cs:328]: Can you please add the reason why we changing this with loc.x as comment

**Review by Sofiya-kumar — DISMISSED** [review]

**Review by rthanga1 — DISMISSED** [review]

File: Trimble.Modus.Components/Controls/Popup/Pages/PopupPage.cs (hunk 1 above)
Comment (amosekr) [reply Trimble.Modus.Components/Controls/Popup/Pages/PopupPage.cs:330]: Fixed

**Review by Sofiya-kumar — APPROVED** [review]

**Pull request merged**